src/components/feed/PostFeed.js

```diff
@@ -131,9 +131,22 @@ class PostFeed extends Component {
   }
 
   handleDateTime(date) {
-    const dateLocal = new Date(date);
-    const timeLocal = dateLocal.toLocaleTimeString();
-    return (String(dateLocal.toDateString()) + " " + timeLocal);
+    const d = new Date(date);
+    const createdDate = d.toDateString();
+    const timeLocal = d.toLocaleTimeString();
+    const de = new Date();
+    const todayLocaltime = de.toLocaleTimeString();
+    const todayDate = de.toDateString();
+    if (todayDate === createdDate) {
+      if (d.getHours() > 12) {
+        return d.getHours() - 12 + " " + "PM";
+      } else {
+        return d.getHours() + " " + "AM";
```

Comment:
`err: Unexpected string concatenation of literals`

Reply:
working on that !